benchmarks/attention/benchmark_attention.py

```diff
@@ -29,8 +27,6 @@
 workspace_opt = True
 # QKV memory layout
 qkv_layout = "bshd_bshd_bshd"
-# sliding window attention
-swa = False
```

Comment:
Why?

Reply:
Because `window_size` is part of the `ModelConfig` now and the tests can figure out whether `swa` is True or not.
https://github.com/cyanguwa/TransformerEngine/blob/cbcea4620cbbeeff565a787b7d902b1ea322a029/tests/pytorch/fused_attn/test_fused_attn.py#L91